src/librustc_expand/proc_macro_server.rs

```diff
@@ -283,6 +285,10 @@ pub struct Group {
     delimiter: Delimiter,
     stream: TokenStream,
     span: DelimSpan,
+    /// A hack used to pass AST fragments to attribute and derive macros
+    /// as a single nonterminal token instead of a token stream.
+    /// FIXME: It needs to be removed, but there are some compatibility issues (see #73345).
+    flatten: bool,
```

Comment:
This is just a boolean equivalent of `FlattenGroup`, right? I think it would be better to use either a `bool` or a `FlattengGroup` consistently.